bedrock/mozorg/templates/mozorg/about/governance/policies/commit/requirements.html

```diff
@@ -0,0 +1,78 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/about-base.html" %}
+
+{% block page_title %}{{ _('Commit Access Requirements') }}{% endblock %}
+{% block body_id %}commit-access-requirements{% endblock %}
+
+{% block article %}
+<h1 class="title-shadow-box">{{ self.page_title() }}</h1>
+
+<p>{{ _('By committing to Mozilla repositories - that is, by directly contributing
+source code, documentation or other data ("Code") to a website or repository
+operated, controlled or managed by the Mozilla Foundation or its affiliates (a "Mozilla
+Repository") - you are indicating your agreement to the terms set out below.') }}</p>
+
+<p>{{ _('For the avoidance of doubt: this document commits you to a set of
+good behaviours; it is not a copyright assignment.') }}</p>
+
+<h2>{{ _('1. Treatment of Account') }}</h2>
```

Comment:
How would that work in a non-hacky way if we wanted to keep them as headings and have the numbers styled the same as the rest of the line?

<ol><li><h2>Title</h2>... wouldn't give the number <h2> styling.
<ol><h2><li>Title</h2>... would mean closing the <h2> before the <li> and misnesting tags.